tests/scattering2d/test_torch_scattering2d.py

```diff
@@ -96,10 +94,9 @@ def test_Modulus(self, backend_device):
 
         y = modulus(x)
         u = torch.squeeze(torch.sqrt(torch.sum(x * x, 3)))
-        v = y.narrow(3, 0, 1)
         u = u.squeeze()
-        v = v.squeeze()
-        assert torch.allclose(u, v)
+        y = y.squeeze()
```

Comment:
Can we say which dimensions are to be squeezed here? Can lead to bad things otherwise.

Reply:
All the dimensions of size 1 are removed, but I can make it explicit.